data/network/src/commonMain/kotlin/social/androiddev/common/network/MastodonApi.kt

```diff
@@ -18,4 +19,24 @@ interface MastodonApi {
      * @see https://docs.joinmastodon.org/methods/instance/
      */
     suspend fun getInstance(domain: String? = null): Result<Instance>
+
+
```

Comment:
Should probably have an auto-formatter running on commit or something (extra line?)

Reply:
Whoops, I will fix. Was hoping Detekt PR gets merged soon